src/main/java/com/softserve/academy/spaced/repetition/DTO/impl/CommentDTO.java

```diff
@@ -18,13 +24,34 @@ public CommentDTO(CourseComment comment, Link link) {
         super(comment, link);
     }
 
-    public Long getCourseCommentId(){return getEntity().getId();}
+    public Long getCommentId() { return getEntity().getId(); }
+
+    public String getCommentText() { return getEntity().getCommentText(); }
+
+    public Date getCommentDate() { return getEntity().getCommentDate(); }
+
+    public String getPersonFirstName() { return getEntity().getPerson().getFirstName(); }
+
+    public String getPersonLastName() { return getEntity().getPerson().getLastName(); }
+
+    public ImageType getImageType() { return getEntity().getPerson().getTypeImage(); }
+
+    public String getImageBase64() { return getEntity().getPerson().getImageBase64(); }
+
+    public String getImage() {
+        return getEntity().getPerson().getImage();
+    }
+
+    public Long getParentCommentId() { return getEntity().getParentCommentId(); }
 
-    public String getCommentText() {return getEntity().getCommentText();}
+    public List<CommentDTO> getListOfChildComments() { return childComments; }
 
-    public Date getCommentDate() {return getEntity().getCommentDate();}
+    public void setChildComments(CommentDTO comment) {
```

Comment:
Please rename to addChildComment. The name of the method should express the method action.